WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -26,10 +26,12 @@ final class OrdersTests: XCTestCase {
     }
 
     func test_create_new_order() throws {
```

Comment:
I know this wasn't introduced in this PR but noticed that this test name is using snake_case instead of camelCase, I think we update it so it's consistent will other UI tests. 

The two new Orders related tests are using this: `test_create_new_order()` and `test_cancel_order_creation()`

Reply:
Just to chime in here, I think we should update the other UI test methods to use snake case (rather than switch these to camel case) to match the project's [naming conventions for test methods](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/naming-conventions.md#test-methods).